### PR TITLE
Partition topology implies strongly Choquet

### PR DESCRIPTION
**Pull request by Moniker1998**
This resolves that S6 is strongly Choquet.

In the proof I've relied on that discrete non-empty space is strongly Choquet, something already known to pi-base:
https://topology.pi-base.org/spaces?q=discrete+%2B+not+empty+%2B+not+strongly+choquet

However I don't have any experience in those game theoretic properties to say more. 
If you have an idea how to modify the proof, feel free to.

### CODE REVIEW

**Comment by Moniker1998** [comment]
@prabau Not "A space $X$"?

File: theorems/T000677.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - P000185: true
+  - P000137: false
+  
+then:
+  P000206: true
+---
+
+A space $X$ is {P206} if and only if the Kolmogorov quotient of $X$ is {P206}.
+
+But if $X$ is {P185} and not {P137} then its Kolmogorov quotient is non-empty and discrete, so strongly Choquet.
```
Comment (david20000813) [inline theorems/T000677.md:14]:
```suggestion
But if $X$ is {P185} and not {P137} then its Kolmogorov quotient is non-empty and {P52}, so strongly Choquet [(Explore)](https://topology.pi-base.org/spaces?q=Discrete+%2B+not+Empty+%2B+not+Strongly+Choquet).
```

Comment (david20000813) [reply theorems/T000677.md:14]: It would be better to have an explore link for a claim like this which is not immediately obvious.

**Comment by david20000813** [comment]
Though, I am slightly frustrated we don't have a property like completely pseudometrizable. That would have implied strongly Choquet. There is one currently available way to generalize this. I'll left it to OP to decide whether to do it: Weakly locally compact + $R_1$ + not Empty => Strongly Choquet. This is because the quotient of a weakly locally compact $R_1$ space is an LCH space, and nonempty LCH spaces are strongly Choquet, see [here](https://topology.pi-base.org/spaces?q=Weakly+locally+compact+%2B+%24T_2%24+%2B+not+Empty+%2B+not+Strongly+Choquet).

**Comment by david20000813** [comment]
The theorem needs to be renumbered due to the just implemented #1121.

**Comment by david20000813** [comment]
Searching through pi-Base again, I think upgrading to Weakly locally compact + $R_1$ + not Empty => Strongly Choquet is useful. It will also imply S54, which does not have a partition topology, is strongly Choquet, a fact that's currently unknown.

**Comment by david20000813** [comment]
@Moniker1998 Steven seems to be away for the moment...? Do you want to make the change to "Weakly locally compact + $R_1$ + not Empty => Strongly Choquet"? Or I can just approve and merge this now if you prefer and leave that change to a later PR?

**Comment by Moniker1998** [comment]
@david20000813 I've already changed it, and yeah, you can approve it if you want.

**Comment by david20000813** [comment]
> @david20000813 I've already changed it, and yeah, you can approve it if you want.

@Moniker1998 Ah, my apologies, I did not notice the change. Approved and merged.

**Review by david20000813 — APPROVED** [review]

**Pull request merged**